main.js

```diff
@@ -128,25 +128,25 @@ app.get('/pay_ticket/:id/:rate', function (req, resp) {
                 var exitTime = enterTime.addHours(req.params.rate);
                 connection.query('UPDATE tickets SET exit_time=? WHERE id=?',[exitTime, req.params.id],
                     function (error2) {
-                   if(error2){
-                       console.log(error2);
-                   }
+                        if(error2){
```

Comment:
I don't think you need `error2` if this was scoped to not have nested callbacks.  Using promises would help solve that issue as well as solve the `connection.query` race conditions that follows this.

Also more descriptive error variable names would help others understand the flow that's intended.